fabric-key-binding-api-v1/src/main/resources/fabric.mod.json

```
@@ -0,0 +1,25 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-key-binding-api-v1",
+  "name": "Fabric Key Bindings API (v1)",
```

```suggestion
  "name": "Fabric Key Binding API (v1)",
```
Also, the name and ID use "key binding" but the description uses "keybinding", should be made consistent.

---

Fixed